services/app-web/src/pages/QuestionResponse/UploadResponse/UploadResponseForm.tsx

```diff
@@ -137,7 +150,7 @@ const UploadResponseForm = ({handleSubmit, apiError, apiLoading, type}: UploadRe
                             variant="default"
                             data-testid="page-actions-right-primary"
                             parent_component_type="page body"
-                            link_url={`/submissions/${id}/rates/${rateID}/question-and-answers?submit=reponse`}
+                            link_url={submitLink}
```

Comment:
**Issue**: The text for this button should be `Submit response`, [Figma](https://www.figma.com/design/frKNnm6lkcpdjJ2eVJE8Bj/Managed-Care-Review?node-id=14541-53262&node-type=frame).